internal/venice-common/src/main/java/com/linkedin/venice/writer/VeniceResourceCloseResult.java

```diff
@@ -0,0 +1,15 @@
+package com.linkedin.venice.writer;
+
+public enum VeniceResourceCloseResult {
+  SUCCESS(0), FAILURE(1), TIMEOUT(2), ALREADY_CLOSED(3);
```

Comment:
Don't find the places where we populate TIMEOUT or FAILURE.

Reply:
It was one of those 'will come in handy in the future' things. I have removed them for now.